packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.test.tsx

```diff
@@ -68,7 +68,7 @@ describe('SelectedPeopleList', () => {
     expect(onItemsRemoved).toBeCalledTimes(1);
   });
 
-  it('group expansion as one of the items in the list', () => {
+  /*it('group expansion as one of the items in the list', () => {
```

Comment:
are you not able to test this because you changed it to async.. or is there any other issue? Did you look into using this - https://www.pluralsight.com/guides/test-asynchronous-code-jest .. if you want to update this test to be simpler , that should be okay to .. something like making sure we are able to edit and see options etc..

Reply:
From what I can tell (and testing) none of that will work. The problem is that we have a UI click which calls a function, which has a promise/then structure. All of those are designed to trust the promise function itself. We could do a test that tests the test GetExpandedItems function to see what it returns, but that wouldn't really test our code

Reply:
yea makes sense.. i would just remove this unit test then.